src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,11 +44,17 @@ public RobotContainer() {
   }
 
   private void configureDefaultCommands() {
+    // Drivebase default command
     Trigger leftBumper = m_driverXboxController.leftBumper();
     Drive driveCommand = new Drive(m_drivebaseSubsystem, m_driverXboxController::getLeftX,
         m_driverXboxController::getLeftY, m_driverXboxController::getRightX, leftBumper::getAsBoolean);
-
     m_drivebaseSubsystem.setDefaultCommand(driveCommand);
+
+    // Claw default command
+    Trigger aButton = m_driverXboxController.a();
+    Trigger bButton = m_driverXboxController.b();
+    ActuateClaw actuationCommand = new ActuateClaw(m_clawSubsystem, aButton::getAsBoolean, bButton::getAsBoolean);
+    m_clawSubsystem.setDefaultCommand(actuationCommand);
```

Comment:
The default command should just be something that stops the claw subsystem.  Create a dedicated StopClawCommand for this purpose.

Reply:
Did this